### ISSUE TITLE
Dropdown's attribute "aria-labelledby" shouldn't be attached if no label is provided

### ISSUE THREAD

**Issue by kmrrakesh** [OfficeDev/office-ui-fabric-react#6545]
### Bug Report 
- __Package version(s)__: ">= 6.0.0 < 7" 
 
- __Browser and OS versions__: Edge and Windows 10

#### Describe the issue:
ARIA attribute, "aria-labelledby" for dropdown is getting attached even if no label is associated with it

It is happening in the latest fabric ui, for all dropdown without associated label, and run keros to replicate it.
It complains as "ARIA attributes must conform to valid values"

#### Actual behavior:
Dropdown is associating an attribute as "aria-labelledby" even if no label is associated

#### Expected behavior:
Dropdown attribute "aria-labelledby" shouldn't be attached if no label is provided

**Comment by Vitalius1** [comment]
@kmrrakesh thanks for submitting this accessibility bug. Could you kindly provide a screenshot of the Keros message after the  FastPass? Also are you scanning the [demo site](https://developer.microsoft.com/en-us/fabric#/components/dropdown) or an application using Dropdown control?
